projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/linked-cases-table/linked-cases-from-table.component.ts

```diff
@@ -0,0 +1,101 @@
+import { LinkedCasesService } from '../../services';
+
+@Component({
+  selector: 'ccd-linked-cases-from-table',
+  templateUrl: './linked-cases-from-table.component.html',
+  styleUrls: ['./linked-cases-from-table.component.scss']
+})
+
+export class LinkedCasesFromTableComponent implements OnInit, AfterViewInit {
+
+  private static readonly CASE_NAME_MISSING_TEXT = 'Case name missing';
+
+  @Input()
+  public caseField: CaseField;
+
+  @Output()
+  public notifyAPIFailure: EventEmitter<boolean> = new EventEmitter(false);
+  public caseDetails: CaseView;
+  public parentUrl: string;
+  public isLoaded: boolean;
+  public getLinkedCasesResponse: CaseLinkResponse[] = [];
+  public linkedCaseReasons: LovRefDataModel[];
+
+  public caseId: string;
+  public showHideLinkText = 'Show';
+  public noLinkedCases = true;
+  public isServerError = false;
+
+  constructor(
+    private readonly route: ActivatedRoute,
+    private readonly casesService: CasesService,
+    public readonly linkedCasesService: LinkedCasesService,
+    ) {
+  }
+
+  public ngAfterViewInit(): void {
+    const labelField = document.getElementsByClassName('case-viewer-label');
+    if (labelField && labelField.length) {
+      labelField[0].replaceWith('');
+    }
+  }
+
+  public ngOnInit(): void {
+    this.fetchPageData();
+    if (this.route.snapshot.data.case) {
+      this.linkedCasesService.caseDetails = this.route.snapshot.data.case;
+    }
+  }
+
+  public fetchPageData() {
+    this.caseId = this.route.snapshot.data.case.case_id;
+    this.getLinkedCases().subscribe(
+      response => {
+        this.isServerError = false;
+        this.getLinkedCasesResponse = response.linkedCases && response.linkedCases.map(item => {
+          const mappedCasetype = this.mapLookupIDToValueFromJurisdictions('CASE_TYPE', item.ccdCaseType);
+          const mappedCasetypeDescription = this.mapLookupIDToValueFromJurisdictions('CASE_TYPE_DESCRIPTION', item.ccdCaseType);
+          const mappedCaseState = this.mapLookupIDToValueFromJurisdictions('STATE', item.state);
+          const mappedCaseStateDescription = this.mapLookupIDToValueFromJurisdictions('STATE_DESCRIPTION', item.state);
+          const mappedCaseService = this.mapLookupIDToValueFromJurisdictions('JURISDICTION', item.ccdJurisdiction);
+          return {
+            ...item,
+            ccdCaseType: mappedCasetype,
+            ccdCaseTypeDescription: mappedCasetypeDescription,
+            ccdJurisdiction: mappedCaseService,
+            state: mappedCaseState,
+            stateDescription: mappedCaseStateDescription,
+            caseNameHmctsInternal: item.caseNameHmctsInternal || LinkedCasesFromTableComponent.CASE_NAME_MISSING_TEXT
+          } as CaseLinkResponse;
+        })
+        this.noLinkedCases = !response.linkedCases || !response.linkedCases.length;
+      },
+      err => {
+        this.isServerError = true;
+        this.notifyAPIFailure.emit(true);
+      }
+      );
```

Comment:
@johnbenjamin-hmcts - check indentation here

Reply:
@Daniel-Lam - I have updated the formatting.